### DISCUSSION TITLE
The Mimir replicating three times the data

### DISCUSSION THREAD

**Discussion by girishms-sentient** [grafana/mimir#6171] [Help and support]
I could see Mimir continuing to replicate data three times.

Running 3 replicas of ingester, 2 distributors, 1 compactor, 1 ruler.

        ################ MIMIR CONFIGURATION #####################
        mimir:
          structuredConfig:
            ingester:
              ring:
                final_sleep: 0s
                num_tokens: 512
                tokens_file_path: /data/tokens
                heartbeat_period: 20s
                heartbeat_timeout: 60s
                unregister_on_shutdown: true
                kvstore:
                  store: memberlist
                replication_factor: 3
                zone_awareness_enabled: false
              instance_limits:
                max_inflight_push_requests: 0

            memberlist:
              abort_if_cluster_join_fails: false
              compression_enabled: false
              join_members:
                - dns+{{ include "mimir.fullname" . }}-gossip-ring.{{ .Release.Namespace }}.svc.{{ .Values.global.clusterDomain }}:{{ include "mimir.memberlistBindPort" . }}
              advertise_addr: ${MY_POD_IP}

            limits:
              ingestion_rate: 200000
              max_global_series_per_metric: 3000000
              max_global_series_per_user: 3001000                
              max_label_names_per_series: 60
              compactor_blocks_retention_period: 0s
              compactor_split_and_merge_shards: 0
              compactor_split_groups: 0

              max_query_parallelism: 224
              request_rate: 700000
              request_burst_size: 50000000
              ingestion_burst_size: 50000000
              out_of_order_time_window: 5m

![image](https://github.com/grafana/mimir/assets/107147378/bc1e9cbc-55ec-409a-a7ce-63c877edc333)

Team, I would appreciate your help.

```[tasklist]
### Tasks
```

**Comment by charleskorn** [comment]
Duplicate of https://github.com/grafana/mimir/discussions/5603#discussioncomment-7143745
